images/converters/ovms/run.sh

```diff
@@ -185,12 +208,20 @@ if [[ "${output_model}" =~ ^/.* ]]; then
 # upload the model remotely, if stored in an S3 bucket
 elif [[ "${output_model}" = "s3://"* ]]; then
     echo "Uploading model to S3 remote storage: ${output_model} ..."
-    mc alias set s3 "${S3_ENDPOINT}" "${AWS_ACCESS_KEY_ID}" "${AWS_SECRET_ACCESS_KEY}"
+    mc alias set s3 "${OUTPUT_S3_ENDPOINT}" "${OUTPUT_AWS_ACCESS_KEY_ID}" "${OUTPUT_AWS_SECRET_ACCESS_KEY}"
```

Comment:
AFAICS `OUTPUT_S3_ENDPOINT` does not default to `s3.amazonaws.com` , is that intentional?